src/page-multi-facility/MultiFacilityPage.tsx

```diff
@@ -1,38 +1,35 @@
 import React from "react";
 import styled from "styled-components";
 
-import PromoBoxWithButton from "../design-system/PromoBoxWithButton";
 import SiteHeader from "../site-header/SiteHeader";
+import ProjectionsHeader from "./ProjectionsHeader";
+import ScenarioSidebar from "./ScenarioSidebar";
 import ToggleRow from "./ToggleRow";
 
-const LeftColumn = styled.div`
-  width: 300px;
-`;
 const MultiFacilityPageDiv = styled.div``;
 
+const MultiFacilityImpactDashboard = styled.main.attrs({
+  className: `
+    h-screen
+    flex
+    mt-8
+  `,
+})``;
+
 const MultiFacilityPage: React.FC = () => {
   return (
     <MultiFacilityPageDiv>
       <div className="font-body text-green min-h-screen tracking-normal w-full">
         <div className="max-w-screen-xl px-4 mx-auto">
           <SiteHeader />
-          <main className="my-6">
```

Comment:
@jovergaag @daschi - I already had a "sidebar" component created locally so I copied the toggle and promo box markup into it.  See the `ScenarioSidebar` component below.